poetry.lock

```diff
@@ -1092,7 +1092,7 @@ diagrams = ["jinja2", "railroad-diagrams"]
 
 [[package]]
 name = "pyright"
-version = "1.1.254"
+version = "1.1.259"
```

Comment:
could you please move this change to separate PR?

Reply:
Oook. I would still separate these two and merge one after another, so that commit history is more lean. Let's do how it's better for you then!

Reply:
Yea I will separate just explained why it's included